packages/toolkit/src/createSlice.ts

```diff
@@ -253,19 +258,16 @@ export function createSlice<
 >(
   options: CreateSliceOptions<State, CaseReducers, Name>
 ): Slice<State, CaseReducers, Name> {
-  const { name, initialState } = options
+  const { name } = options
   if (!name) {
     throw new Error('`name` is a required option for createSlice')
   }
+  const initialState =
```

Comment:
:hand: Doesn't this interfere with the lazy init logic I just added to `createReducer`?  We should be able to just pass this through.

Reply:
Since `createReducer` will be called later than it was before, I wanted to keep the freezing behaviour as soon as possible. It should not really interfere with it, the "work" is just being done twice.